PurchasesTests/Purchasing/ProductInfoTests.swift

```diff
@@ -105,7 +105,7 @@ class ProductInfoTests: XCTestCase {
         let productInfo: ProductInfo = .createMockProductInfo(discounts: [discount1, discount2, discount3])
         
         expect(productInfo.asDictionary()["offers"] as? [[String: NSObject]]).toNot(beNil())
-        guard let receivedOffers = productInfo.asDictionary()["offers"] as? [[String: NSObject]] else { fatalError() }
+        let receivedOffers = try XCTUnwrap(productInfo.asDictionary()["offers"] as? [[String: NSObject]])
```

Comment:
this is an improvement, because if `XCTUnwrap` fails, it'll throw and show as a test failure, but it'll allow other tests in the suite to run. `fatalError` will halt execution instead.